fenix/app/src/main/java/org/mozilla/fenix/shopping/ShoppingExperienceFeature.kt

```diff
@@ -0,0 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.shopping
+
+import org.mozilla.fenix.nimbus.FxNimbus
+
+/**
+ * An abstraction for shopping experience feature flag.
+ */
+class ShoppingExperienceFeature {
+
+    /**
+     * Returns true if the shopping experience feature is enabled.
+     */
+    fun isEnabled() = FxNimbus.features.shoppingExperience.value().enabled
```

Comment:
drive by: if you prefer `val` usages over functions, you could instead make this a val with a `get` override

Reply:
It's fair, we don't need any params and both compile to the same code, updated!